.ossls.yml

```diff
@@ -1,10 +1,10 @@
 gomod:
   mod-file: go.mod
 
-yarn:
-  lockfile: ui/apps/platform/yarn.lock
-  node-modules-dirs:
-    - ui/apps/platform/node_modules
+# TODO re-enable this and test with updated ossls tool
```

Comment:
Do you have a ROX-12345 ticket for this follow-up?

Reply:
I don't, I was hoping to get the ossls change in with this PR so that we don't have a sequence of builds missing the third party notices for UI dependencies.

I've got the ossls changes merged and a new release created. We just need https://github.com/stackrox/rox-ci-image/pull/212 and https://github.com/stackrox/rox-ci-image/pull/211 merged I believe to get the new binary into the CI images.

Once all those are merged I'll update the reference to the build image in stackrox/stackrox and uncomment the lines in this file.